### PR TITLE
bump whylabs-client to 0.6.0

### PR DESCRIPTION
**Pull request by murilommen**
## Description
Updating whylabs-client to reflect latest changes on the API

- [x] I have reviewed the [Guidelines for Contributing](CONTRIBUTING.md) and the [Code of Conduct](CODE_OF_CONDUCT.md).

### CODE REVIEW

File: python/pyproject.toml [hunk 1]
```diff
@@ -16,7 +16,7 @@ whylogs-sketching = ">=3.4.1.dev3"
 protobuf = ">=3.19.4"
 importlib-metadata = { version = "<4.3", python = "<3.8" }
 typing-extensions = {version = ">=3.10", markers = "python_version < \"4\""}
-whylabs-client = "^0.5.10"
+whylabs-client = "0.6.0"
```
Comment (jamie256) [inline python/pyproject.toml]: we could use the carrot there so we don't need to update whylogs on each patch release.

**Review by jamie256 — DISMISSED** [review]
LGTM!

File: python/pyproject.toml (hunk 1 above)
Comment (murilommen) [reply python/pyproject.toml]: added back :)

**Review by jamie256 — APPROVED** [review]
LGTM!

**Pull request merged**